test/server.js

```diff
@@ -60,5 +60,23 @@ server.get('/metrics', (req, res) => {
   res.end(register.metrics());
 });
 
-console.log('Server listening to 3000, metrics exposed on /metrics endpoint');
+server.get('/protractor', (req, res) => {
+  var sys = require('util');
+  var exec = require('child_process').exec;
+  var child;
+  console.log("[INFO] Running protractor tests");
+  child = exec("protractor /protractor/protractor.conf.js", function (error, stdout, stderr) {
+    // sys.print('stderr: ' + stderr);
+    if (error !== null) {
+      console.log('ERROR: ' + error);
+    }
+    console.log(stdout);
+  });
+  res.end();
+});
+
```

Comment:
Nice! Shall we `[info]` prefix these?  I dunno how much we care about consistency for this.  I did the same with the `/helpers/logger.js` [logger](https://github.com/benjaminapetersen/origin-web-console-smoke-test/pull/12/files#diff-1d35ece61a2a4349acf21a9db5efa932) & added a little color...prob the color was unnecessary, heh.

Reply:
yeah lets prefix them. Although not sure if we want to he in upper case `[INFO]` or lowercase `[info]`.. since the `lib/log/output.sh` is prefixing with the uppercase.

Reply:
Good catch, i guess match?